src/lava/lib/optimization/utils/generators/mis.py

```diff
@@ -0,0 +1,180 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+# See: https://spdx.org/licenses/
+
+
+import numpy as np
+import numpy.typing as npt
+import networkx as netx
+
+from lava.lib.optimization.problems.problems import QUBO
+
+
+class MISProblem:
+    """
+    Utility class to instantiate Maximum Independent Set problems and
+    convert them to a QUBO formulation.
+    """
+
+    def __init__(self, num_vertices: int, connection_prob: float,
+                 seed: int = None):
+        """
+        Instantiate a new random MIS problem, with the given number of
+        vertices and connection probability.
+
+        :param num_vertices: Number of vertices of the random graph

```

Comment:
I think our standard is...
`Parameters
----------
parameter_1: type
    Description`

Reply:
Should we remove python type hints then?